styles/scss/apie.module.scss

```diff
@@ -1,114 +1,78 @@
-                font-size: 1.6em;
-                padding-bottom: 15px;
-            }
-            h4 {
-                font-weight: bold;
-                font-size: 1.5em;
-                padding-bottom: 10px;
-            }
-        }
+    h1 {
+      font-weight: bold;
+      font-size: 2em;
+      padding-bottom: 30px;
     }
-}
-
-@media screen and (max-width: 2000px) {
-    .innerDiv {
-        .authorImg {
-            width: 25vw !important;
-        }
-        .authorBio {
-            width: 30vw !important;
-            padding-left: 0px !important;
-            font-size: 1em !important;
-        }
+    h2 {
+      font-weight: bold;
+      font-size: 1.8em;
+      padding-bottom: 20px;
+    }
+    h3 {
+      font-weight: bold;
+      font-size: 1.6em;
+      padding-bottom: 15px;
     }
+    h4 {
+      font-weight: bold;
+      font-size: 1.5em;
+      padding-bottom: 10px;
+    }
+  }
 }
 
-@media screen and (max-width: 1600px) {
-    .innerDiv {
-        .authorImg {
-            width: 30vw !important;
-        }
-    }
+@media screen and (min-width: $breakpoint-mobileLandscape) {
+  .authorInfo {
+    padding: 100px 10%;
+  }
 }
 
-@media screen and (max-width: 1200px) {
-    .innerDiv {
-        .authorImg {
-            all: unset;
-        }
-    }
+@media screen and (min-width: $breakpoint-tablet) {
+  .authorImg {
+    margin: auto;
+    padding-bottom: 100px;
+    width: 70%;
+  }
+  .authorBio {
+    width: 70%;
+  }
 }
 
-@media screen and (max-width: 1000px) {
-    .innerDiv {
-        padding-top: 100px !important;
-        .authorInfo {
-            flex-direction: column;
-            .authorImg {
-                padding: 0 !important;
-                width: 70% !important;
-                margin-bottom: 50px;
-            }
-            .authorBio {
-                width: 100% !important;
-                padding: 0px 15% 20% 15% !important;
-            }
-        }
+@media screen and (min-width: $breakpoint-tabletLandscape) {
+  .authorInfo {
+    all: unset;
+    min-height: 100vh;
+    margin: auto;
+    padding-top: 100px;
+    display: flex;
+    justify-content: space-between;
+    width: 70%;
+
+    .authorImg {
+      margin: 0;
     }
+    .authorBio {
+      margin-right: 0;
+    }
+  }
 }
 
-@media screen and (max-width: 650px) {
-    .innerDiv {
-        padding-top: 150px !important;
-        .authorInfo {
-            flex-direction: column;
-            .authorImg {
-                padding: 0 !important;
-                width: 100% !important;
-                margin-bottom: 50px;
-            }
-            .authorBio {
-                // width: 100% !important;
-                padding: 20px !important;
-            }
-        }
-    }
+@media screen and (min-width: $breakpoint-desktop) {
+  .authorInfo {
+    max-width: $breakpoint-full;
+  }
+  .authorImg {
+    width: 25vw !important;
+  }
+  .authorBio {
+    margin: 0;
+    padding: 0;
+    width: 30vw !important;

```

Comment:
Venk !important kaip velnias kryziaus
https://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css

Pasidomek kaip viekia css specificity https://css-tricks.com/specifics-on-css-specificity/